<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSeriesDialog</class>
 <widget class="QDialog" name="EditSeriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit series</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblStartChunk">
       <property name="text">
        <string>Start chunk</string>
       </property>
       <property name="toolTip">
        <string>Index of start geometry chunk</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="sbStartChunk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblEndChunk">
       <property name="text">
        <string>End chunk</string>
       </property>
       <property name="toolTip">
        <string>Index of end geometry chunk</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbEndChunk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblInterval">
       <property name="text">
        <string>Interval</string>
       </property>
       <property name="toolTip">
        <string>Distance between geometry chunks included in this series</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="sbInterval">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lblField">
       <property name="text">
        <string>Field</string>
       </property>
       <property name="toolTip">
        <string>Field of geometry chunks to modify</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cbField">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="lblStartValue">
       <property name="text">
        <string>Start value</string>
       </property>
       <property name="toolTip">
        <string>Initial field value (default 0)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="leStartValue"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="lblEndValue">
       <property name="text">
        <string>End value</string>
       </property>
       <property name="toolTip">
        <string>Field value to approach (optional)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="leEndValue"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="lblIncrement">
       <property name="text">
        <string>Increment</string>
       </property>
       <property name="toolTip">
        <string>Amount to alter field value each iteration (default 0)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="leIncrement"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
